### PR TITLE
New Featured Artist: SOOOO

### PR DESCRIPTION
**Pull request by Ephemeralis**
👀 plz

---

Welcome SOOOO to the Featured Artist lineup!

### CODE REVIEW

File: news/2018-11-30-new-featured-artist-soooo.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "New Featured Artist: SOOOO"
+date: 2018-11-30 20:00:00 +0000
+---
+
+The Featured Artist project storms into the sounds of a turbulent youth with the latest addition to the roster, **SOOOO!**
```
Comment (Toybickler) [inline news/2018-11-30-new-featured-artist-soooo.md]: to its roster

File: news/2018-11-30-new-featured-artist-soooo.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+[![](https://assets.ppy.sh/artists/30/header.jpg)](https://osu.ppy.sh/beatmaps/artists/30)
+
+First popularized on osu! following the tremendous success of their track *Happppy song*, **SOOOO** is an enigmatic Japanese Vocaloid producer that thrives on an unusual subversion of the classical poppy, sweet-tinged sound in favor of a more intense, experimental, and ultimately sinister twist.
+
+Dense, intimidating and full of musical influences from 'heavier' genres, **SOOOO**'s sound is not for the faint of heart in many regards - mapping included.
```
Comment (TPGPL) [inline news/2018-11-30-new-featured-artist-soooo.md]: italic `heavier` instead?

File: news/2018-11-30-new-featured-artist-soooo.md (hunk 1 above)
Comment (Ephemeralis) [reply news/2018-11-30-new-featured-artist-soooo.md]: i'm kind of hesitant to change this, 'to its roster' feels wrong/worse here?

anyone else weigh in?

Comment (cl8n) [reply news/2018-11-30-new-featured-artist-soooo.md]: "its" makes more sense cuz "featured artist project" is the subject of sentence

Comment (Ephemeralis) [reply news/2018-11-30-new-featured-artist-soooo.md]: teams don't really say "add to its roster" though, they say "add to THE roster"

which is sort of what i'm gunning for

**Pull request merged**